apiary.apib

```diff
@@ -103,6 +103,14 @@ the [table list](#reference/tables/create-or-list-tables/tables-in-bucket) also
 the table bucket properties and table attributes. You can use the `include` parameter
 to return only the required resources in the response.
 
+## HTTP headers
+Some API calls supports custom http headers.
+
+Supported HTTP headers are:
+- `X-StorageApi-Token (string)`: storage api authentication token
+- `X-KBC-RunId (string)`: id mainly used to match component run with storage job
+- `X-KBC-Backend (json string)`: header used to customize underlying backend configuration. Supported options are `context` and `size` (Supported option are different for each backend). Example: `'{"context":"custom-wlm-group","size":"XL"}'`
```

Comment:
```suggestion
- `X-KBC-Backend (json string)`: header used to customize underlying backend configuration. Supported options are `context` and `size` (Supported option are different for each backend). Example: `'{"context":"custom-wlm-group"}'` or `'{"size":"XL"}'`
```

Reply:
S pohledu API je to jedno funguje obo za ráz to jestli to podporuje oboje backend je druhá věc. Dokážu si představit, že to funguje dohromady. Třeba na SNFLK nastavíš podle size warehouse a podle context nastavíš třeba query tag.

Reply:
OK